bin/ch-test

```diff
@@ -697,28 +812,45 @@ scope_int=$(scope_to_digit "$CH_TEST_SCOPE")
 # Execute phase
 case $phase in
     build)
+        builder_check
         dir_tar_mk
-        test_make_auto
+        test_make build
         test_build
         win
         ;;
     run)
         dir_img_mk
-        test_make_auto
+        test_make
```

Comment:
It might be clearer for `test_make` to always or never take an argument.

Reply:
That was a bug, good catch.